docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/intellij-13-code-style.md

```diff
@@ -9,7 +9,8 @@ nav_order: 13
 Contributions to JabRef's source code need to have a code formatting that is consistent with existing source code. For that purpose, JabRef provides code-style and check-style definitions.
 
 Install the [CheckStyle-IDEA plugin](http://plugins.jetbrains.com/plugin/1065?pr=idea), it can be found via the plug-in repository:
-Navigate to **File > Settings... > Plugins"**.
+Navigate to **File > Settings... > Plugins"**.<br>
```

Comment:
I hink, the line "On the top" will directly come after `"File".` This is not intended.

Reply:
The flow in Windows/old macs: `File`->`Settings`->`Plugins`->`Marketplace`...
The flow in M2 macs: `IntelliJ`->`Settings`->`Plugins`->`Marketplce`...
Only change is the first action's location, which had to be alternatively pointed out for m2 macs below, rest is the same. So I think the line in the middle should stay.

Reply:
OK I think I get what you mean now, just checked the preview.
Yes, it's better if we move the middle line (13) to after 16.